tsdb/head_append.go

```diff
@@ -982,8 +988,7 @@ func (a *headAppender) Commit() (err error) {
 		series.Unlock()
 	}
 
-	histogramsTotal := len(a.histograms)
-	histoOOORejected := 0
+	histogramsAppended := len(a.histograms)
```

Comment:
Nit: I find it a little inconsistent to only move histoOOORejected, but not histogramsAppended. So I'd put the initialization of histogramsAppended to `len(a.histograms)+len(a.floatHistograms)` on top as well and remove the assignments here and on line 1017(new).

Reply:
Updated in https://github.com/prometheus/prometheus/pull/13607/commits/841a133514297cfeb2ea09277d168d1c7d107815